src/main/java/org/terasology/manualLabor/systems/ShearingSystem.java

```diff
@@ -148,6 +148,8 @@ private void switchPrefab(EntityRef entity, String meshURI, String materialURI)
             if (skeletalMesh.isPresent() && shearedMaterial.isPresent()) {
                 skeletalMeshComponent.mesh = skeletalMesh.get();
                 skeletalMeshComponent.material = shearedMaterial.get();
+            } else {
+                logger.warn("Model switch fail due to invalid assets");
```

Comment:
:+1: for adding the warning. Consider what other information you might want in this message to help you debug it, e.g. the URIs it was attempting to switch to.